components/common/ShareButtons.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+const ShareButtons = ({ className }) => (
+    <div className={ className }>
+        <button type="button" className="btn btn-sm btn-primary mr-2">
+            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
+        </button>
+        <button type="button" className="btn btn-sm btn-primary" style={{backgroundColor: '#1b95e0', borderColor: '#1b95e0'}}>
+            <FontAwesomeIcon icon={ faTwitter } className="mr-1" /> Tweet
```

Comment:
These are icon fonts, right? Will using these components cause accessibility issues? If so, we should make note of that

Reply:
Checked with VoiceOver, no issues 👍

Reply:
Cool, yea, this is another case where I copy pasted from `ModalButton.js`, so glad to see that it works OK!

https://github.com/RagtagOpen/freefrom-map-frontend/blob/c76018945051f37e7a8ff7266e0f51c6c0d5a88c/components/modal/ModalButton.js#L10-L12